crawl-ref/source/tiledoll.cc

```diff
@@ -286,11 +286,12 @@ void fill_doll_equipment(dolls_data &result)
         tileidx_t ch;
         switch (you.species)
         {
-        case SP_CENTAUR: ch = TILEP_TRAN_STATUE_CENTAUR;  break;
```

Comment:
It's not hugely important, but in general, it's better not to align items like this; it hinders using tools like git-blame. Probably best if you just add the faerie dragon line and let it stick out the side.